### ISSUE TITLE
v2.2.2 produces PaxHeader compilation errors when used with Berkshelf and Librarian

### ISSUE THREAD

**Issue by kainosnoema** [sous-chefs/redisio#147]
### tl;dr

Packaging a site cookbook on OS X 10.6+ using `bsdtar`, with files containing extended attributes, causes runtime compilation errors for end-users extracting using Ruby's Minitar (e.g. Berkshelf and Librarian): https://tickets.opscode.com/browse/CHEF-4107
### Details:

As of v2.2.2, we're getting runtime compilation errors like the following:

```
================================================================================
Recipe Compile Error in /var/folders/dy/tblyd6dn1jdbsglnk9kp60cm0000gn/T/d20140823-83378-hstg7l/cookbooks/redisio/providers/PaxHeader/configure.rb
================================================================================

SyntaxError
-----------
/var/folders/dy/tblyd6dn1jdbsglnk9kp60cm0000gn/T/d20140823-83378-hstg7l/cookbooks/redisio/providers/PaxHeader/configure.rb:1: syntax error, unexpected tIDENTIFIER, expecting end-of-input
16 gid=72107521
      ^

Cookbook Trace:
---------------
  /Users/evan/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/chef-11.12.8/lib/chef/mixin/from_file.rb:42:in `class_eval'
  /Users/evan/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/chef-11.12.8/lib/chef/mixin/from_file.rb:42:in `class_from_file'
  /Users/evan/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/chef-11.12.8/lib/chef/provider/lwrp_base.rb:94:in `build_from_file'
  /Users/evan/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/chefspec-4.0.1/lib/chefspec/extensions/chef/lwrp_base.rb:29:in `build_from_file_with_removal'
```

According to the [Jira ticket mentioned above](https://tickets.opscode.com/browse/CHEF-4107), "this is an edge-case caused by these conditions":
- OS X 10.6+ as the cookbook packager
- Cookbook files which contain extended attributes
- Extraction through Ruby's Minitar (used by Berkshelf and Librarian)

My guess is that v2.2.2 introduced a file that contains extended attributes. The best way to fix this is to package using `gnutar` instead of `bsdtar` and resubmitting an update. This has affected many other cookbooks, including `artifact`: RiotGames/artifact-cookbook#86.
